template-manager-tag/src/main/java/com/sequenceiq/cloudbreak/tag/DefaultCostTaggingService.java

```diff
@@ -47,9 +59,8 @@
             LOGGER.debug("The tag template is: {}", entry.getValue());
             String generatedValue = centralTagUpdater.getTagText(preparationObject, entry.getValue());
             LOGGER.debug("The generated tag value is: {}", generatedValue);
-            addAccountTag(result, entry.getKey(), generatedValue, request.getPlatform());
```

Comment:
addAccountTag is unused now, right?

Reply:
so this is the extracted method. If you check upper there is a used method

Reply:
Right!